tests/test_kessel.py

```
@@ -37,19 +37,24 @@ def test_subject_from_identity_no_user_id(identity):
 
 
-def test_host_groups_for(mocker):
+@pytest.mark.asyncio
+async def test_host_groups_for(mocker):
     responses = [
         mocker.Mock(object=mocker.Mock(resource_id="grp-1")),
         mocker.Mock(object=mocker.Mock(resource_id="grp-2")),
     ]
-    list_workspaces = mocker.patch("roadmap.kessel.list_workspaces", return_value=responses)
+
+    async def _fake_list(*args, **kwargs):
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `_fake_list`

(ANN202)